packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1197,43 +1245,51 @@ const getFullModel = <TChannelFactory extends IChannelFactory, TOperation extend
 	);
 
 /**
- * Runs only the provided seeds.
- *
- * @example
- *
- * ```typescript
- * // Runs only seed 42 for the given model.
- * createDDSFuzzSuite.only(42)(model);
- * ```
+ * {@inheritDoc (createDDSFuzzSuite:function)}
+ * @internal
  */
-createDDSFuzzSuite.only =
```

Comment:
Note for reviewers: this syntax implicitly creates a namespace called `createDDSFuzzSuite` under the hood, which we were not exporting, which caused API-Extractor to complain.